packages/utils/eslint-config-custom/package.json

```diff
@@ -6,8 +6,19 @@
   "dependencies": {
     "@babel/eslint-parser": "^7.19.1",
     "@strapi/eslint-config": "0.1.2",
+    "@typescript-eslint/eslint-plugin": "^5.55.0",
```

Comment:
Ha nice catch 👍

Reply:
Should we remove them from the root level too I'm not sure it's done

Reply:
Yep, these eslint deps are isolated to this package 👍🏼